src/v2/Apps/Auction/Routes/ConfirmBid/index.tsx

```diff
@@ -186,7 +176,14 @@ export const ConfirmBidRoute: React.FC<ConfirmBidProps> = props => {
           address_zip: address.postalCode,
         }
 
-        const token = await createTokenFromAddress(stripeAddress)
+        const { error, token } = await stripe.createToken(stripeAddress)
+
+        if (error) {
+          actions.setFieldError("creditCard", error.message)
```

Comment:
So this means that if a payment is declined (or some other tokenization level error happens) – the user will see that reason? If so, that's awesome. What's the expected behavior after this PR on the registration form? Will any sort of tokenization-level error be displayed there as well?

Reply:
> What's the expected behavior after this PR on the registration form?

The user should see an error message from Stripe after clicking the submit button, which is what you can see in the screenshot. There are a number of errors that could occur in this instance (missing input, Stripe service not available, etc).